packages/firestore/src/api/database.ts

```diff
@@ -1116,7 +1116,7 @@ export class DocumentReference implements firestore.DocumentReference {
     options: ListenOptions,
     observer: PartialObserver<firestore.DocumentSnapshot>
   ): Unsubscribe {
-    let errHandler = (err: Error) => {
+    let errHandler = (err: Error): void => {
```

Comment:
Is there a way to turn off the return-type enforcement for very simple functions?

Reply:
The rule doesn't provide an option to do that currently.
An alternative is 
```javascript
type ErrorHandler = (err: Error) => void;
let errHandler: ErrorHandler = (err) => {
...
}
```

Reply:
I vote option #1 then (leave as is in the current PR).

Reply:
[The TSLint rule](https://palantir.github.io/tslint/rules/typedef/) had an option to ignore arrow functions, which I was using in Messaging and Installations. If this rule has a similar option, I think it's safe to enable that.